ERP systems section update (#7)
ezsystems/commerce-documentation: docs/guide/erp_integration/checkout_order/order_submission/order_submission.md

For example it is not possible to use the `CustomerProfileDataService` (`ses.customer_profile_data.ez_erp`), and the Repository (`ezpublish.api.repository`) must be used instead in order to fetch the user data.

## Sending additional data in the order

You can extend the message with additional fields in two ways:

- Extend the `CreateSalesOrderMessage` class. This solution requires maintenance effort in case of an update of the shop's base implementation, as changes to the base message must be merged to the extended version.
- Write the data into [one of the SesExtension](../../erp_communication/erp_components\erp_component_messages/erp_message_calculatesalesorder_createsalesorder.md) [tree elements (see `ses_tree` in table)](../../erp_communication/erp_components\erp_component_messages/erp_message_class_generator.md)).
This solution should only be used to add simple, single-valued fields.
During the communication with the remote service (e.g. Web.Connector), the tree attributes are mapped between PHP arrays and XML data implicitly without any structure definition.
This can cause conflicts for XML elements that occur multiple times, and sequential PHP arrays.

### Example

> Sometimes it is necessary to send additional (project-specific data) in the order.

Reviewer: ```suggestion
Sometimes it is necessary to send additional project-specific data in the order.
```

> You can get this information from basket (`dataMap`) or from customer data and need to send it to ERP.

Reviewer: ```suggestion
If you need to send the data to ERP, you can get it from basket (`dataMap`) or from customer data.
```
